DataFormats/EgammaCandidates/interface/Conversion.h

```diff
@@ -35,6 +35,8 @@ namespace reco {
 			      arbitratedEcalSeeded=1, 
 			      arbitratedMerged=2,
 			      arbitratedMergedEcalGeneral=3,
+			      gsfTracksOnly=4,
```

Comment:
This ```gsfTracksOnly``` looks like never being used.
Is it a remnant of some never finalized development, a placeholder for some future update, or what?

Reply:
Yes, it is indeed a remnant of a private try I had made. Will clean that up